sendrecv/js/webrtc.js

```diff
@@ -165,20 +165,17 @@ function onServerError(event) {
 }
 
 function getCamera() {
-    var constraints;
-    var textarea = document.getElementById('constraints');
     try {
-        constraints = JSON.parse(textarea.value);
+         JSON.parse(constraints.value);
```

Comment:
I was puzzled that this would even work, but some quick googling told me both that it does, and that using this feature isn't exactly good practice (https://coderwall.com/p/umiruq/javascript-getelementbyid-not-required). Why do you want to do that?

Reply:
More clean and readable code. Sometimes it is surprising, but only on the beginning.
IMHO (and not only mine) less code is better.

Reply:
Sorry, but in our opinion readable code is ***always*** better than "less code". We will not accept any changes that make code harder to understand, especially since this is supposed to be an example for other people to read and understand.

Reply:
I removed controversial commits. Would you like to review the rest?

Reply:
The "rename" commit still makes use of that "feature", though in a slightly more explicit way. Please remove it too and I'll merge the charset commit, this one is clearly correct.

Reply:
though please wrap your commit message to have 80 columns max :)

Reply:
Actually 'rename' it was not using that feature at all. It was just used the same name for DOM and var. Anyway, removed.